<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyLabelImgClass</class>
 <widget class="QMainWindow" name="MyLabelImgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyLabelImg</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="8,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="browseLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1">
        <item>
         <widget class="QPushButton" name="prevButton">
          <property name="text">
           <string>prev</string>
          </property>
          <property name="shortcut">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyQLabel" name="imgLabel">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextButton">
          <property name="text">
           <string>next</string>
          </property>
          <property name="shortcut">
           <string>Right</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="saveLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>label:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLabelLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="useLabelButton">
        <property name="text">
         <string>use</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyQLabel</class>
   <extends>QLabel</extends>
   <header location="global">MyQLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyLabelImg.qrc"/>
 </resources>
 <connections/>
</ui>
